### PR TITLE
Add autostore support

### PR DESCRIPTION
**Pull request by dmitry-mukhin**

### CODE REVIEW

**Comment by dmitry-mukhin** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;3aEuoPPd&#x2F;102-pyuploadcare-store"> Pyuploadcare: добавить store</a>

**Comment by homm** [comment]
wip?

**Comment by dmitry-mukhin** [comment]
nope

File: pyuploadcare/api_resources.py [hunk 1]
```diff
@@ -297,11 +352,36 @@ def upload_from_url(cls, url):
         return file_from_url
 
     @classmethod
-    def upload_from_url_sync(cls, url, timeout=30, interval=0.3,
-                             until_ready=False):
+    def upload_from_url_sync(cls, url, store=None, filename=None, timeout=30,
```
Comment (homm) [inline pyuploadcare/api_resources.py]: breaking change with no reasons.

`upload_from_url_sync(cls, url, timeout, interval, until_ready, store, filename)`

File: .travis.yml [hunk 2]
```diff
@@ -37,6 +33,8 @@ matrix:
   include:
     - python: 2.7
       env: DJANGO=1.8 SUITE="tests/integration"
+  allow_failures:
+    - env: DJANGO=1.10
```
Comment (dmitry-mukhin) [inline .travis.yml]: had no time to investigate, something with new django admin or smth

File: pyuploadcare/api_resources.py (hunk 1 above)
Comment (dmitry-mukhin) [inline pyuploadcare/api_resources.py]: +1

File: tests/integration/test_api_resources.py [hunk 3]
```diff
@@ -252,12 +244,11 @@ def test_successful_store(self):
...
 
 
-class FileCopyTest(unittest.TestCase):
+class FileCopyTest(RestAPITestCase):
 
     @classmethod
     def setUpClass(cls):
-        conf.pub_key = 'demopublickey'
-        conf.secret = 'demoprivatekey'
+        super(FileCopyTest, cls).setUpClass()
```
Comment (dmitry-mukhin) [inline tests/integration/test_api_resources.py]: why

Comment (homm) [inline tests/integration/test_api_resources.py]: noop

Comment (dmitry-mukhin) [inline tests/integration/test_api_resources.py]: Setting of API keys moved to parent class. This test case needs them.

Comment (homm) [inline tests/integration/test_api_resources.py]: oh, ok, missed long tail

**Pull request merged**